src/hooks/useGrowthbookGetFeatureValue/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react';
 import { Analytics } from '@deriv-com/analytics';
-import useIsGrowthbookIsLoaded from './useIsGrowthbookLoaded';
+import useIsGrowthbookIsLoaded from '../useIsGrowthbookLoaded';
```

Comment:
is this changes intended?

Reply:
Yes, previously it was a file in `src/hooks/useGrowthbookGetFeatureValue.tsx`, moved it to a folder for consistency with other hooks